app/subcommands/group/run

```diff
@@ -6,16 +6,15 @@ set -eu
 
 [ -n "${1:-}" ] || fatality 'must provide a group name to add too'
 group_name="$1"
-client_variant="${2:-}"
-
+shift
 run_with_dependencies() {
 	depcmd="$1"
 	configkey="$2"
-	clientvariant="${3:-}"
+	shift 2
 	configpath="$(config_path "$configkey")"
 	[ -f "$configpath" ] || return 0
-	xargs -I{} sh -c "$depcmd {} $clientvariant" <"$configpath"
+	xargs -I{} sh -c "$depcmd {} ${@:-}" <"$configpath"
```

Comment:
I am not certain but I think you will want `$*` here instead of `$@` as the former works in the middle of a string where the latter is meant to be the entire string iirc.